analytics/terraform/emr-eks-karpenter/data.tf

```diff
@@ -2,6 +2,10 @@ data "aws_eks_cluster_auth" "this" {
   name = module.eks.cluster_name
 }
 
+data "aws_ecr_authorization_token" "token" {
+  registry_id = "895885662937"
```

Comment:
Can you not use the below ecr authentication which is used in Karpenter as well to pull the images?

Reply:
I can't use it because the below is only for ECR public, EMR on EKS use its own private ECRs